lectures/inventory_ssd.md

```diff
@@ -0,0 +1,216 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Inventory Management Model
+
+```{include} _admonition/gpu.md
+```
+
+```{code-cell} ipython3
+!pip install quantecon
+```
+
+```{code-cell} ipython3
+import quantecon as qe
+import jax
+import jax.numpy as jnp
+import numpy as np
+import matplotlib.pyplot as plt
+from collections import namedtuple
+```
+
+Let's check the GPU we are running
+
+```{code-cell} ipython3
+!nvidia-smi
+```
+
+```{code-cell} ipython3
+jax.config.update("jax_enable_x64", True)
+```
+
+```{code-cell} ipython3
+# NamedTuple Model
+Model = namedtuple("Model", ("K", "c", "κ", "p", "z_vals", "Q"))
+```
+
+```{code-cell} ipython3
+@jax.jit
+def demand_pdf(p, d):
+    return (1 - p)**d * p
+```
+
+```{code-cell} ipython3
+def create_sdd_inventory_model(
+        ρ=0.98, ν=0.002, n_z=20, b=0.97,   # Z state parameters
+        K=40, c=0.2, κ=0.8, p=0.6):        # firm and demand parameters
+    mc = qe.tauchen(n_z, ρ, ν)
+    z_vals, Q = jnp.array(mc.state_values + b), jnp.array(mc.P)
+    # rL = jnp.max(jnp.abs(jnp.linalg.eigvals(z_vals * Q)))
```

Comment:
This is because:
```
Nonsymmetric eigendecomposition is only implemented on the CPU backend. If your matrix is symmetric or Hermitian, you should use eigh instead.
NotImplementedError: Nonsymmetric eigendecomposition is only implemented on the CPU backend. If your matrix is symmetric or Hermitian, you should use eigh instead.
```